docs/azure/GettingStarted/Deploy.rst

```diff
@@ -67,36 +71,58 @@ Deploy the solution to your account
       az provider register --namespace Microsoft.AzureActiveDirectory
 
 #. Create an Azure Active Directory B2C: currently, it is not possible to create Active Directory B2C and application through the ARM template (see `GitHub issue <https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/issues/1>`_).
-   You must follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=applications>`_ and create a B2C tenant and an application.
+
+   1. You must follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=applications>`_ and create a B2C tenant and an application.
    Use ``http://localhost:3000/`` (for local development) and ``https://<your APP_NAME>app.z16.web.core.windows.net/`` as the redirect URLs.
 
-#. Save the initial domain name of the created Active Directory B2C to the environment variable ``B2C_TENANT``
+   #. Enable the *implicit grant and hybrid flows* for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
+
+#. Create the user flow for sign up, sign in, and make sure to name the userflow as ``B2C_1_signup_signin``.
```

Comment:
```suggestion
#. Create the user flow for sign-up, sign-in, and make sure to name the userflow as ``B2C_1_signup_signin``.
```
If and when these are used as nouns, you need to have the hyphen. No hyphen, when used as verbs.